pkg/storage/storage.go

```diff
@@ -54,23 +58,23 @@ func (p *storage) GetNodeMetrics(nodes ...string) ([]api.TimeInfo, []corev1.Reso
 
 	timestamps := make([]api.TimeInfo, len(nodes))
 	resMetrics := make([]corev1.ResourceList, len(nodes))
-
 	for i, node := range nodes {
 		metricPoint, present := p.nodes[node]
 		if !present {
 			continue
 		}
 
+		prevMetricPoint := p.prevNodes[node]
```

Comment:
If previousMetricPoint is empty then we will continue and pass it to `cpuUsageOverTime`. Looking in this function we will exit on first if `prevMetricPoint.Timestamp.IsZero()`, thus resulting in usage being reported as `0`?

I think it would be more preferential to skip this node to avoid reporting wrong value.

Reply:
You are very right, in the first place I though it would be preferential to still report the memory usage, but after giving it another thought, I prefer the approach where we skip the resource completely.